src/util/api/likernft/fiat/stripe.ts

```
@@ -94,10 +93,10 @@ export async function processStripeFiatNFTPurchase(session, req) {
     wallet = await findWalletWithVerifiedEmail(email);
   }
   const isPendingClaim = !wallet;
-  let claimToken;
+  let claimToken: string | undefined;
   if (isPendingClaim) {
     wallet = LIKER_NFT_PENDING_CLAIM_ADDRESS;
-    claimToken = randomBytes(32).toString('base64url');
+    claimToken = docData.claimToken;
```

Not sure if we need to check if `docData.claimToken` exists, need to throw an error maybe

---

Added guard